src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -173,15 +174,68 @@ Copyright (c) .NET Foundation. All rights reserved.
     package references before NuGet reads them.
     ============================================================
   -->
-  <Target Name="CollectPackageReferences" Returns="@(PackageReference)" />
+  <Target Name="CollectPackageReferences" Returns="@(PackageReference)" >
+    <!-- NOTE for design-time builds we need to ensure that we continue on error. -->
+    <PropertyGroup>
+      <CollectPackageReferencesContinueOnError>$(ContinueOnError)</CollectPackageReferencesContinueOnError>
+      <CollectPackageReferencesContinueOnError Condition="'$(ContinueOnError)' == '' ">false</CollectPackageReferencesContinueOnError>
+    </PropertyGroup>
+
+    <CheckForDuplicateNuGetItemsTask
+      Condition="'$(DisableCheckingDuplicateNuGetItems)' != 'true' "
```

Comment:
What is the default value of `DisableCheckingDuplicateNuGetItems` property? If it is `false` then this task will execute because the condition evaluates to true.

Reply:
There's no default, so it's ``.